### DISCUSSION TITLE
Method to look up the local IP address

### DISCUSSION THREAD

**Discussion by wlandau** [wlandau/crew#95] [Announcements]
`crew` needs to get the local IP address of the client R session so workers can dial into the address over TCP. I am using `getip::getip()` for this, and it works on all the machines I can access, but it is not usable in cases like https://github.com/wlandau/crew.cluster/issues/1#issuecomment-1546024163. In addition, `getip` is limited to IPv4, it can't do IPv6 yet.

**Comment by shikokuchuo** [comment]
The issue is there is not a single way of getting this - it is highly platform dependent. Hence, probably (i) see if the maintainers of `getip` are open to extending their package or (ii) wrap system calls (although this again will be different on each platform).
